_data/locales/es.yml

```diff
@@ -12,6 +12,8 @@ sections:
   libraries: Librerías
   appendix: Appendix
   contributors: Contributors
+  translation_report: Translation Report
+  section: Section
```

Comment:
I would name this translation_report_dection as if we ever use section in other part of the website it may have a different meaning in another languagredepending on the context

Reply:
Update: lets move this under translation_report list